packages/lib/services/searchengine/SearchEngineUtils.ts

```diff
@@ -49,9 +50,24 @@ export default class SearchEngineUtils {
 		// issue: https://discourse.joplinapp.org/t/how-to-recover-corrupted-database/9367
 		if (noteIds.length !== notes.length) {
 			// remove null objects
-			return sortedNotes.filter(n => n);
+      const results2:any = [];
```

Comment:
The formatting here is off, wonder how it got past the linter

Reply:
I think the check failed because of this